lib/queuePopulator/KafkaLogConsumer/LogConsumer.js

```diff
@@ -0,0 +1,166 @@
+const kafka = require('node-rdkafka');
+const async = require('async');
+const ListRecordStream = require('./ListRecordStream');
+
+// maximum time to wait for consumer group to rebalance (ms)
+const maxDelayRebalance = 120000;
```

Comment:
why "2 minute", is this some common/standard value?
* If we rely on some default value, can't we just leave the field empty and let kafka use whatever value it finds appropriate (or has in its config_
* otherwise, should this be added to the config?

Reply:
2 minutes is just a safe value that i found while testing, it usually takes less time than that to rebalance.
I don't think there is a use for this being in the config, and the value is used in a custom function that waits for rebalance, so there is no default Kafka value for that